demos/python_demos/im2latex_demo/utils.py

```diff
@@ -0,0 +1,32 @@
+import pickle as pkl
+import json
+
+START_TOKEN = 0
+PAD_TOKEN = 1
+END_TOKEN = 2
+UNK_TOKEN = 3
```

Comment:
Should these values come from `--vocab_path` file instead of being hard coded?

Reply:
Actually all these four values are the same (I mean indices) in every vocab file since they are hard coded in file which creates vocab.  That is true that `PAD_TOKEN` and `UNKNOWN_TOKEN` are note used in demo, they are only used while training the model. We can delete them. As for start & end token, they can be passed via args, but again, they are always 0 and 1 respectively, so we can fix them as constants in demo or in the utils.py (as it is done now)

Reply:
+1 for deleting unused constants.

Reply:
Deleted